src/profile-logic/profile-data.js

```diff
@@ -1759,25 +1759,10 @@ export function convertStackToCallNodePath(
  *
  * If no samples are found, 0 is returned.
  */
-export function computeCallNodeMaxDepth(
-  thread: Thread,
-  callNodeInfo: CallNodeInfo
-): number {
-  let maxDepth = 0;
-  const { samples } = thread;
-  const { callNodeTable, stackIndexToCallNodeIndex } = callNodeInfo;
-  for (let i = 0; i < samples.length; i++) {
-    const stackIndex = samples.stack[i];
-    if (stackIndex !== null) {
-      const callNodeIndex = stackIndexToCallNodeIndex[stackIndex];
-      // Change to one-based depth
-      const depth = callNodeTable.depth[callNodeIndex] + 1;
-      if (depth > maxDepth) {
-        maxDepth = depth;
-      }
-    }
-  }
-  return maxDepth;
+export function computeCallNodeMaxDepth(callNodeInfo: CallNodeInfo): number {
+  if (callNodeInfo.callNodeTable.depth.length === 0) return 0;
```

Comment:
Thanks for checking this edge case!

On the other hand, we try to avoid adding `if` blocks without brackets in the project. Because it's mostly hard to read and easy to make something wrong with these type of `if`s :) Can you please add brackets that contain `return 0;`?

Reply:
Yes, it has been updated now!